spec/swagger.json

```diff
@@ -0,0 +1,522 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Officer Filing API",
+    "version": "0.0.1"
+  },
+  "host": "api.chs.local",
```

Comment:
I wonder if the host here should be the live one `api.companieshouse.gov.uk`
see https://github.com/companieshouse/confirmation-statement-api/blob/main/spec/schema.json#L8

Reply:
Good spot, i'll update it.